src/coretelephony.cs

```diff
@@ -80,8 +82,29 @@ interface CTTelephonyNetworkInfo {
 		CTCarrierEventHandler CellularProviderUpdatedEventHandler { get; set; }
 #endif
 
+		[Deprecated (PlatformName.iOS, 12,0, message: "Use 'ServiceCurrentRadioAccessTechnology' instead.")]
 		[iOS (7,0), Export ("currentRadioAccessTechnology")]
 		NSString CurrentRadioAccessTechnology { get; }
+
+		[iOS (12,0)]
+		[NullAllowed]
+		[Export ("serviceSubscriberCellularProviders", ArgumentSemantic.Retain)]
+		NSDictionary<NSString, CTCarrier> ServiceSubscriberCellularProviders { get; }
+
+		[iOS (12,0)]
+		[NullAllowed]
+		[Export ("serviceCurrentRadioAccessTechnology", ArgumentSemantic.Retain)]
+		NSDictionary<NSString, NSString> ServiceCurrentRadioAccessTechnology { get; }
+
+		[iOS (12,0)]
+		[NullAllowed]
+		[Export ("serviceSubscriberCellularProvidersDidUpdateNotifier", ArgumentSemantic.Copy)]
+		Action<NSString> CellularProvidersDidUpdateNotifier { get; set; }
```

Comment:
This is inconsistent with other APIs. It should be `ServiceSubscriberCellularProvidersDidUpdateNotifier`.

Reply:
_Note: don't forget to update the deprecated message if you change this._

Reply:
The swift signature, if it's of any help, is long: https://developer.apple.com/documentation/coretelephony/cttelephonynetworkinfo/3024512-servicesubscribercellularprovide

Reply:
the name has not been swift-inized - but you're right about consistency :)